atd-vze/src/views/Locations/Location.js

```diff
@@ -24,25 +24,13 @@ function Location(props) {
 
   const [variables, setVariables] = useState({
     id: locationId,
-    yearsAgoDate: fiveYearsAgo,
-    costPerCrash: parseFloat(0),
+    yearsAgoDate: fiveYearsAgo
   });
 
   const { loading, error, data, refetch } = useQuery(GET_LOCATION, {
     variables,
   });
 
-  // Retrieve est_comp_cost_amount of Non-CR3 crashes from DB and set updated variable
-  useEffect(() => {
-    if (
-      Object.entries(data).length !== 0 &&
-      data.locationTotals[0].noncr3_est_comp_cost === 0
-    ) {
-      const costPerNonCr3Crash = data.nonCr3EstCompCost[0].est_comp_cost_amount;
-      setVariables({ ...variables, costPerCrash: costPerNonCr3Crash });
-    }
-  }, [data, variables]);
-
```

Comment:
This is a brilliant removal, thank you @frankhereford for walking me through it 🙌